### PR TITLE
docs(troubleshooting): fix instructions on how to download the reset uf2

### PR DESCRIPTION
**Pull request by okke-formsma**
I couldn't find the "clear" (now called reset) uf2 file. This updates the docs.

### CODE REVIEW

**Review by innovaker — CHANGES_REQUESTED** [review]
Couple of suggestions.  I can't test this procedure as I don't use uf2.

File: docs/docs/troubleshooting.md [hunk 1]
```diff
@@ -54,7 +54,9 @@ Since then, a much simpler procedure of performing a bluetooth reset for split k
 
 **New Procedure:**
 
-1. Log into Github and download the "settings clear" UF2 image from the [latest build in Github Actions](https://github.com/zmkfirmware/zmk/actions?query=workflow%3ABuild+branch%3Amain)
+1. Log into Github and click on [latest build in Github Actions](https://github.com/zmkfirmware/zmk/actions?query=workflow%3ABuild+branch%3Amain) (it's the first 'result' on that page)
```
[the remark this answers is not in the document]
Comment (mcrosson) [reply docs/docs/troubleshooting.md:57]: I took it quite literally myself, update looks good

File: docs/docs/troubleshooting.md [hunk 2]
```diff
@@ -54,7 +54,9 @@ Since then, a much simpler procedure of performing a bluetooth reset for split k
...
+1. From the next page under "Artifacts", download the "`board name`-settings_reset-zmk" zip file.
```
[the remark this answers is not in the document]
Comment (mcrosson) [reply docs/docs/troubleshooting.md:58]: Agreed. That also is a more consistent approach with other forms of documentation when something is a variable. You could also use `[` and `]` which are common in technical documents.

**Comment by okke-formsma** [comment]
@innovaker @mcrosson Sorry I somehow missed your comments. Updated and pushed.

**Review by Nicell — APPROVED** [review]
LGTM

File: docs/docs/troubleshooting.md [hunk 3]
```diff
@@ -54,7 +54,9 @@ Since then, a much simpler procedure of performing a bluetooth reset for split k
 
 **New Procedure:**
 
-1. Log into Github and download the "settings clear" UF2 image from the [latest build in Github Actions](https://github.com/zmkfirmware/zmk/actions?query=workflow%3ABuild+branch%3Amain)
+1. Open [latest build in Github Actions](https://github.com/zmkfirmware/zmk/actions?query=workflow%3ABuild+branch%3Amain) and open the first 'result' on that page.
```
Comment (innovaker) [inline docs/docs/troubleshooting.md:57]:
```suggestion
1. [Open the GitHub `Actions` tab and select the `Build` workflow](https://github.com/zmkfirmware/zmk/actions?query=workflow%3ABuild+branch%3Amain).
1. Select the top 'result' on that page.
```

File: docs/docs/troubleshooting.md [hunk 4]
```diff
@@ -54,7 +54,9 @@ Since then, a much simpler procedure of performing a bluetooth reset for split k
...
+1. From the next page under "Artifacts", download the `$boardname-settings_reset-zmk` zip file.
+1. Unzip the downloaded file
```
Comment (innovaker) [inline docs/docs/troubleshooting.md:59]:
```suggestion
1. Unzip the downloaded file.
```

**Review by innovaker — COMMENTED** [review]
Was about to merge this when I spotted these on a final read.  I'll squash them in if you accept them.

File: docs/docs/troubleshooting.md (hunk 4 above)
Comment (mcrosson) [reply docs/docs/troubleshooting.md:59]: I'm good with all of the above changes. Do I need to do anything with these?

**Comment by okke-formsma** [comment]
looks good, thanks @innovaker

**Review by innovaker — APPROVED** [review]

**Pull request merged**